nixos/modules/services/web-apps/stash.nix

```diff
@@ -0,0 +1,95 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.services.stash;
+in
+{
+  options = {
+    services.stash = {
```

Comment:
It seems like there is a configuration file, but that they recommend most of the configuration to be done through the web interface. Maybe we should create a freeform `settings` option for this config file? `ip` (`host`) and `port` would have to be moved inside `settings`.

https://docs.stashapp.cc/advanced-tutorials/advanced-configuration-options/